test/tests.js

```diff
@@ -8,3 +8,4 @@ require('../packages/graphql-server-hapi/dist/hapiApollo.test');
 require('../packages/graphql-server-koa/dist/koaApollo.test');
 require('../packages/graphql-server-restify/dist/restifyApollo.test');
 require('../packages/graphql-server-express/dist/apolloServerHttp.test');
+require('../packages/graphql-server-lambda/dist/lambdaApollo.test');
```

Comment:
can you take that 1 line upper? :) apolloServerHttp is not integration test all modules are running but extra testing..